packages/bpk-component-rating/src/BpkRating.js

```diff
@@ -172,23 +144,21 @@ const BpkRating = (props: Props) => {
 
 BpkRating.propTypes = {
   ariaLabel: PropTypes.string.isRequired,
-  title: PropTypes.string.isRequired,
+  title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
   value: PropTypes.number.isRequired,
   className: PropTypes.string,
   ratingScale: PropTypes.oneOf(Object.keys(RATING_SCALES)),
   size: PropTypes.oneOf(Object.keys(RATING_SIZES)),
   subtitle: PropTypes.string,
-  type: PropTypes.oneOf([RATING_TYPES.default, RATING_TYPES.pill]),
-  vertical: PropTypes.bool,
+  showScale: PropTypes.bool,
 };
 
 BpkRating.defaultProps = {
   className: null,
-  ratingScale: RATING_SCALES.zeroToTen,
+  ratingScale: RATING_SCALES.zeroToFive,
   size: RATING_SIZES.base,
   subtitle: null,
-  type: RATING_TYPES.default,
-  vertical: false,
+  showScale: false,
```

Comment:
I feel like this should always be `true` by default instead of `false` as from the designs and stories it only looks like the TA examples might sometimes not show the scale so it should be opt-in at that point instead of the other way around

Reply:
Yeah, most of cases `showScale` is true except for Trip Advisor and one individual rating. It also make sense for setting default value to `true` which other cases and using the component by passing `false`